static/app/actionCreators/sentryAppTokens.tsx

```diff
@@ -35,16 +35,16 @@ export async function addSentryAppToken(
  *
  * @param {Object} client ApiClient
  * @param {Object} app SentryApp
- * @param {String} token Token string
+ * @param {String} token_id Id of the token
  */
 export async function removeSentryAppToken(
   client: Client,
   app: SentryApp,
-  token: string
+  token_id: string
```

Comment:
@cathteng guessing I should change this too right? To `tokenId`?

Reply:
ah yes

Reply:
done!